main.py

```diff
@@ -20,6 +20,16 @@ def suutiit_zinju():
   dati = request.json
   chats.pieraksti_zinju(dati)
   return chats.lasi()
+  
+@app.route('/chats/get_mood')
+def sanjemt_zinju():
+  return chats.lasi_mood()
+
+@app.route('/chats/set_mood', methods=['POST'])
```

Comment:
Šis jau izskatās pēc "saglabāt garastāvokli",
varētu action: "/garastavokli/saglabat" un metodi:
```python
   saglabat_garastavokli
```